### PR TITLE
dev-util/android-tools: add 34.0.5

### PR DESCRIPTION
**Pull request by Jamim**
_Hello everyone,_

**android-tools** 34.0.5 [was released](https://github.com/nmeum/android-tools/releases/tag/34.0.5) recently. It would be nice to have it in the Portage tree.

Based on:
  - [android-tools-34.0.1::gentoo](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-util/android-tools/android-tools-34.0.1.ebuild)
  - [android-tools-34.0.4::pg_overlay](https://gitlab.com/Perfect_Gentleman/PG_Overlay/-/blob/master/dev-util/android-tools/android-tools-34.0.4.ebuild)

<details>
<summary>diff -u android-tools-34.0.1.ebuild android-tools-34.0.5.ebuild</summary>

```diff
--- android-tools-34.0.1.ebuild	2024-03-07 22:57:34.483843095 +0300
+++ android-tools-34.0.5.ebuild	2024-03-23 00:29:09.631007359 +0300
@@ -3,9 +3,9 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_COMPAT=( python3_{10..12} )
 
-inherit flag-o-matic cmake python-r1
+inherit cmake python-r1
 
 DESCRIPTION="Android platform tools (adb, fastboot, and mkbootimg)"
 HOMEPAGE="https://github.com/nmeum/android-tools/ https://developer.android.com/"
@@ -19,7 +19,7 @@
 # The entire source code is Apache-2.0, except for fastboot which is BSD-2.
 LICENSE="Apache-2.0 BSD-2"
 SLOT="0"
-KEYWORDS="amd64 ~arm arm64 x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 IUSE="python udev"
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
@@ -46,8 +46,6 @@
 
 src_prepare() {
 	eapply "${DISTDIR}/${PN}-31.0.3-no-gtest.patch"
-	eapply "${FILESDIR}/${PN}-34.0.0-protobuf.patch"
-	eapply "${FILESDIR}/${PN}-34.0.1-include-algorithm.patch"
 
 	cd "${S}/vendor/core" || die
 	eapply "${S}/patches/core/0011-Remove-the-useless-dependency-on-gtest.patch"
@@ -61,11 +59,13 @@
 }
 
 src_configure() {
-	# bug 905713
-	use elibc_musl && append-flags -D_LARGEFILE64_SOURCE
-
 	local mycmakeargs=(
 		# Statically link the bundled boringssl
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		-DCMAKE_FIND_PACKAGE_PREFER_CONFIG=ON \
+		-Dprotobuf_MODULE_COMPATIBLE=ON
 		-DBUILD_SHARED_LIBS=OFF
 	)
 	cmake_src_configure
```
</details>

_Best regards!_

### CODE REVIEW

**Comment by Jamim** [comment]
_Hello @perfect7gentleman,_
I suppose this PR might be interesting to you since it's based on your work.

File: dev-util/android-tools/metadata.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		<changelog>https://android.googlesource.com/platform/packages/modules/adb/+log</changelog>
 		<doc>https://android.googlesource.com/platform/packages/modules/adb/+/HEAD/OVERVIEW.TXT</doc>
 		<remote-id type="cpe">cpe:/a:google:android_debug_bridge</remote-id>
-		<remote-id type="github">android/platform_system_core</remote-id>
+		<remote-id type="github">nmeum/android-tools</remote-id>
 		<remote-id type="launchpad">ubuntu</remote-id>
```
Comment (Jamim) [inline dev-util/android-tools/metadata.xml:10]: By the way, does this line mean anything?

**Comment by perfect7gentleman** [comment]
> _Hello @perfect7gentleman,_ I suppose this PR might be interesting to you since it's based on your work.

Thanx, I'll check and test it.

File: dev-util/android-tools/metadata.xml (hunk 1 above)
Comment (zmedico) [reply dev-util/android-tools/metadata.xml:10]: Thanks, removed in 79429e7d0a3d11445044344e4e4c67a3e5aee130.

**Comment by zmedico** [comment]
Thanks!

**Pull request closed without merging**